docs/index.md

```diff
@@ -43,88 +43,61 @@ Mouse includes enrichments for:
 
 #### Example:
 
-```scala
-scala> import mouse.all._
+```scala mdoc
 import mouse.all._
 
-scala> true.option("Its true!")
-res0: Option[String] = Some(Its true!)
+true.option("Its true!")
```

Comment:
@armanbilge Sorry, I mean that comments with results of function calls could be helpful.
```suggestion
true.option("Its true!")
// res0: Option[String] = Some(Its true!)
```

Reply:
I'm sorry, I still couldn't understand 😕 

@danicheg the site looks like this right now.
https://8000-armanbilge-mouse-vcuwbgovp34.ws-us29.gitpod.io/

Reply:
Wow, shame on me, I forgot how `mdoc` works. Thanks!